src/ContaoCommunityAlliance/DcGeneral/Contao/View/Contao2BackendView/EditMask.php

```diff
@@ -644,7 +644,8 @@ public function execute()
         $blnSubmitted            = ($inputProvider->getValue('FORM_SUBMIT') === $definition->getName());
         $blnIsAutoSubmit         = ($inputProvider->getValue('SUBMIT_TYPE') === 'auto');
 
-        $widgetManager = new ContaoWidgetManager($environment, $this->model);
+        $encoder       = new PropertyValueEncoder($environment);
```

Comment:
I'm not very happy with the instantation of the encoder. I prefer defining it as service. But as the encoder only triggers the event there should not be any reasons to define another one. So I could live with that solution.